testsuite/features/step_definitions/command_steps.rb

```diff
@@ -944,6 +941,28 @@
   $server.run(cmd)
 end
 
+When(/^I install "([^"]*)" product on the proxy$/) do |product|
+  out, = $proxy.run("zypper ref && zypper --non-interactive install --auto-agree-with-licenses --force-resolution -t product #{product}")
+  STDOUT.puts "Installed #{product} product: #{out}"
+end
+
+When(/^I install proxy pattern on the proxy$/) do
+  if $product == 'Uyuni'
```

Comment:
It might be more Ruby style, if we use those oneliners..
`pattern = $product == 'Uyuni' ? 'uyuni_proxy' : 'suma_proxy'`

But ok, could be more readable as it is.

Reply:
no you are right, I will change that

Reply:
done